Zend/tests/typehints/or_null.phpt

```diff
@@ -219,99 +219,99 @@ try {
 
 ?>
 --EXPECTF--
-TypeError: Argument 1 passed to unloadedClass() must be an instance of I\Dont\Exist or null, instance of stdClass given, called in %s/or_null.php on line 8 and defined in %s/or_null.php:5
+TypeError: Argument 1 passed to unloadedClass() must be an instance of I\Dont\Exist or null, instance of stdClass given, called in %s on line 8 and defined in %s:5
 Stack trace:
-#0 %s/or_null.php(8): unloadedClass(Object(stdClass))
+#0 %s(8): unloadedClass(Object(stdClass))
```

Comment:
I usually prefer to replace the `/` with `%e` (no need to change the code, though :-)).